### DISCUSSION TITLE
SOAPClient

### DISCUSSION THREAD

**Discussion by Lavignotte** [docker-library/wordpress#785] [General]
Hi,

I've been using this Docker image in production for 4 years now and I am very happy with it, so thank you to the contributors team !

Now, I need a new Wordpress extension (for WooCommerce indeed) that has a depency with SoapClient.
It is not installed by default in the Apache image and I failed to install it myself in a child image (the package is not available with apt).

Anyone nows how to install it ?

Thank you

**Comment by tianon** [comment]
https://github.com/docker-library/wordpress/issues/708#issuecomment-1112435460 :eyes:
